.github/workflows/alphanet_openzeppelin_test.yml

```diff
@@ -0,0 +1,48 @@
+name: gw_alphanet_v1_openzeppelin_test
+
+
+on:
+  schedule:
+  - cron:  '50 20/24 * * *' # every day
```

Comment:
[`50 20/24 * * *`](https://crontab.guru/#50_20/24_*_*_*) may not work with every cron.